.changes/next-release/enhancement-awscrt-51540.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "awscrt",
+  "description": "Update minimum awscrt version requirement to 0.23.1"
```

Comment:
Nit: also add that we replaced cryptography with CRT for CloudFront URL signing

Reply:
Oops! In rebasing I lost my feature changelog entry. Thanks for catching that.

Reply:
I'm imagining we may need to adjust or combine all of these when merging the feature branch later anyway.

Reply:
Yeah, agree. I'll leave this out for now and will write a single feature update on the `cryptography-to-crt` branch.